src/task1-1.js

```diff
@@ -1,18 +1,22 @@
-const readline = require("readline");
+const readline = require('readline');
+
 const rl = readline.createInterface({
   input: process.stdin,
-  output: process.stdout
+  output: process.stdout,
 });
 
-askQuestion();
-
 function reverse(str) {
-  return str.split("").reverse().join("")
+  return str
```

Comment:
this function returns immediately - you can convert it to arrow function

Reply:
Noted and updated. Linter split it into lines, but I'll always prefer arrow functions when then fit.